File: src/main/java/com/cleverlance/workshop/Pythagoras.java
package com.cleverlance.workshop;

public class Pythagoras {

    public static void main(String[] args) {
        int a = 3;
        int b = 4;
        int c = calc(a, b);
        System.out.println(c);
    }

    static int calc(int a, int b) {
        return (int) Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2));
    }
}

File: src/main/java/com/cleverlance/workshop/Pythagoras.kt
package com.cleverlance.workshop

import kotlin.math.pow
import kotlin.math.sqrt

fun main() {
        val a = 3
        val b = 4
        val c = calc(a, b)
        println(c)
    }

    fun calc(a: Int, b: Int): Int {
        return sqrt(a.toDouble().pow(2) + b.toDouble().pow(2)).toInt()
}